src/absil/illib.fs

```diff
@@ -672,42 +672,44 @@ let _ = eventually { use x = null in return 1 }
 //---------------------------------------------------------------------------
 // generate unique stamps
 //---------------------------------------------------------------------------
-
-type UniqueStampGenerator<'T when 'T : equality>() = 
-    let encodeTab = new Dictionary<'T,int>(HashIdentity.Structural)
-    let mutable nItems = 0
-    let encode str = 
-        if encodeTab.ContainsKey(str)
-        then
-            encodeTab.[str]
-        else
-            let idx = nItems
-            encodeTab.[str] <- idx
-            nItems <- nItems + 1
-            idx
-    member this.Encode(str)  = encode str
+[<Sealed>]
+type UniqueStampGenerator<'T when 'T : equality>(?initialCapacity) =
```

Comment:
Please remove the initialCapacity argument since it is not used in your code diff